### PR TITLE
obo:BFO -> bfo:BFO obo:IAO -> iao:IAO

### PR DESCRIPTION
**Pull request by andrecastro0o**

### CODE REVIEW

**Comment by andrecastro0o** [comment]
@StroemPhi  wait for me to add a test, before you merge

**Comment by andrecastro0o** [comment]
Changes:
*  prefix obo: was replace with the prefix of the corresponding obo foundry ontologies:  bfo, iao, ro, obi, ico, ncbitaxon, gaz, cro
* test `def test_obo_ontos_resolution` was introduced to test IRI obo foundry ontologies
* IAO ontology IRI was unpinned  from a specific version

File: aeon.ttl
Comment (andrecastro0o) [inline aeon.ttl:5]: This is where IAO was pinned to a specif version @StroemPhi

**Review by andrecastro0o — COMMENTED** [review]
:+1:

File: aeon.ttl
Comment (andrecastro0o) [reply aeon.ttl:5]: Hence bfo prefix will become
`@prefix bfo: <https://standards.iso.org/iso-iec/21838/-2/ed-1/en/owl/bfo-2020.owl> .`

**Review by StroemPhi — APPROVED** [review]

**Review by andrecastro0o — COMMENTED** [review]
:+1:

**Review by andrecastro0o — COMMENTED** [review]
:+1:

**Pull request merged**
